### DISCUSSION TITLE
Allow dismissing resource save business rule errors

### DISCUSSION THREAD

**Discussion by maxpatiiuk** [specify/specify7#5719] [Feature Requests]
To prevent loss of data.
Or at very least turn the table into read only mode but still make it accessible

**Comment by maxpatiiuk** [comment]
This was implemented in the agent merging dialog
Need to test whether it's safe and if so, do the same everywhere else where the Save button is present

**Comment by grantfitzsimmons** [comment]
@melton-jason @realVinayak @acwhite211 

Please close if this has been resolved.
